third-party/realdds/doc/DDS ICD.md

```diff
@@ -1,29 +1,32 @@
+# Topics
+
+Information in realdds domain is being passed using 4 main topics
 * **realdds/device-info** of type flexible - The server sends a sample of this topic to inform readers of a connected camera.
-  Some of the details included are the camera model and serial number, this information will be used in the other topic names to ease communication with verious cameras in the network.
-* **realdds/\<model\>/\<serial\>/notification** of type flexible - Notifications are sent from the camera server to the camera user.
-  Some notificaitons are being sent at discovery to convey the full device information, e.g streams, supported options, etc...
+  Some of the details included are the *camera model* and *serial number*, this information will be used in the other topic names to ease communication with various cameras in the network.
+* **realdds/\<model\>/\<serial\>/notification** of type flexible - Notifications are sent from the camera's server to the camera's user.
+  Some notifications are being sent at discovery to convey the full device information, e.g. streams, supported options, etc...
   Other notifications are sent as needed, mostly as reply to a control message
-* **realdds/\<model\>/\<serial\>/control** of type flexible - Controls are commands from the camera user to the camera, e.g start streaming, set option value etc...
-* **realdds/\<model\>/\<serial\>/<stream name>** of type image - Frames of data of the appropriate stream from the camera server to the user.
+* **realdds/\<model\>/\<serial\>/control** of type flexible - Controls are commands from the camera's user to the camera, e.g. start streaming, set option value etc...
+* **realdds/\<model\>/\<serial\>/\<stream name\>** of type image - Frames of the selected stream from the camera's server to the user.
+
+All the topics in the realdds domain use one of these 2 topic types
+* **flexible** - Carries a buffer of data that can be JSON, CBOR or other user custom data. See [flexible.idl](https://github.com/IntelRealSense/librealsense/blob/dds/third-party/realdds/include/realdds/topics/flexible/flexible.idl).
+* **image** - A buffer of data with some extra fields that describe the image. See [image.idl](https://github.com/IntelRealSense/librealsense/blob/dds/third-party/realdds/include/realdds/topics/image/image.idl).
+
+For instructions about how to generate code from the IDL files see [here](https://github.com/IntelRealSense/librealsense/blob/dds/third-party/realdds/include/realdds/topics/readme.md).
 
 # Messages
 
 ## device-info
 
-On the camera user side, `realdds::dds_device_watcher` can be used to listen to incoming device-info messages.
+On the camera's user side, `realdds::dds_device_watcher` can be used to listen to incoming device-info messages.
 When compiling librealsense with the `BUILD_WITH_DDS` flag the context automatically creates and uses a `realdds::dds_device_watcher` object to identify DDS connected cameras.
```

Comment:
Maybe you can add that this is a CMake flag. (When compiling librealsense with the CMake flag BUILD...)

Reply:
Added